### ISSUE TITLE
Timepicker is too large for smaller devices

### ISSUE THREAD

**Issue by damiancioo** [vanpra/compose-material-dialogs#81]
When using timepicker, for example

```
MaterialDialog = MaterialDialog().also {
    it.build {
        timepicker(
            title = "Select time",
            initialTime = LocalTime.now(),
            onTimeChange = {}
        )

        buttons {
            positiveButton(
                res = android.R.string.ok
            )
            negativeButton(
                res = R.string.cancel_button_title
            )
        }
    }
}
```
On smaller devices (for example Nexus S 480x800), whole dialog is a little too big, so buttons are not rendering to user.

There might be added configuration for adjusting timepicker elements size.

![device-2021-06-09-131505](https://user-images.githubusercontent.com/37381435/121345549-47c50800-c925-11eb-909f-e5fe8c156e04.png)

**Comment by nglauber** [comment]
[Almost] the same applies to Date dialog... Buttons are not visible... 

![Screenshot_1623424097](https://user-images.githubusercontent.com/6018094/121708344-d8c6eb00-caad-11eb-8c79-06dec99d8373.png)

**Comment by vanpra** [comment]
Thanks for letting me know about this. This is likely due to the updated the top and bottom padding on the dialog to better follow the material design spec. I'll take a look at how I can scale the rest of the dialog contents to account for the smaller screen size.

**Comment by damiancioo** [comment]
it's exactly how it was on 480x800 phones :/

**Comment by vanpra** [comment]
Should be fixed in 0.5.0 release
